tests/src/unit_v2_tests.cpp

```diff
@@ -215,4 +215,56 @@ TEST(stronk_units_v2, default_unit_can_do_all_the_usual_stuff)
     EXPECT_GT(val, -val);
 }
 
-}  // namespace twig::unit_v2::unit_tests
+struct A : unit<A, can_equate>
+{
+};
+struct B : unit<B, can_equate>
+{
+};
+struct C : unit<multiplied_dimensions_t<A, B>, can_equate>
+{
+};
```

Comment:
Say I wanted to add a custom consteval constructor for ´C::value<some_specific_representation>` - can I do that directly now? Or would I need to implement a skill that conditionally adds such a constructor for the type in question?